crates/vm/levm/src/call_frame.rs

```diff
@@ -94,12 +134,13 @@ impl CallFrame {
         self.pc
     }
 
-    pub fn jump(&mut self, jump_address: U256) {
+    /// Jump to the given address, returns false if the jump position wasn't a JUMPDEST
+    pub fn jump(&mut self, jump_address: U256) -> bool {
         if !self.valid_jump(jump_address) {
-            // Should be a halt when we implement it
-            panic!("Invalid jump");
+            return false;
         }
-        self.pc = jump_address.as_usize();
+        self.pc = jump_address.as_usize() + 1;
```

Comment:
```suggestion
        self.pc = jump_address.as_usize();
```
Why jump to the `address + 1` instead of just `address`? I'm guessing that in the way as this is implemented right now the opcode `JUMPDEST` will be skipped. I know it doesn't do anything but I think that a more correct approach would be to jump into that location and then if the VM sees that the current opcode is `JUMPDEST` it just goes to the next instruction.

Reply:
done!